ui/packages/catalog-ui-search/src/main/webapp/component/map-info/map-info.view.js

```diff
@@ -62,8 +63,21 @@ module.exports = Marionette.LayoutView.extend({
     this.$el.toggleClass('has-feature', this.model.get('target') !== undefined)
   },
   serializeData: function() {
+    const that = this
     let modelJSON = this.model.toJSON()
+    let summaryModel = {}
+    const summary = props.summaryShow
+    summary.forEach(function(attribute) {
+      if (that.model.get('targetMetacard') !== undefined)
+        summaryModel[attribute] = that.model
```

Comment:
We might need to massage some types of data to make them look how the user expects (dates, etc.).

Reply:
One other thing to handle would be multi value attributes and thumbnails.

Reply:
👍